static_report/src/components/ComparisonReport/CRColumnsSummary.tsx

```diff
@@ -0,0 +1,34 @@
+import { Text, Icon } from '@chakra-ui/react';
+import { FiArrowUpCircle, FiArrowDownCircle } from 'react-icons/fi';
+
+export function CRColumnsSummary({
+  base,
+  target,
+}: {
+  base: number;
+  target: number;
+}) {
+  if (base === target) {
+    return (
+      <>
+        <Text as="span">{base}</Text>
+        <Text as="span">{'->'}</Text>
```

Comment:
can we use a react icon here?
Depending on font-family, this may look different